build/build.sh

```diff
@@ -10,19 +10,17 @@ set -ex
 main() {
     CARGO="$(builder)"
 
-    if is_linux && is_musl; then
-        # jemalloc doesn't allow 16K page sizes for linux musl 
-        # https://github.com/microsoft/ripgrep-prebuilt/issues/26
-        JEMALLOC_SYS_WITH_LG_PAGE=14
-    fi
-
     # Test a normal debug build.
     if is_arm || is_aarch64 || is_ppc64le; then
         "$CARGO" build --target "$TARGET" --release --features 'pcre2'
     # pcre2 is not supported on s390x
     # https://github.com/zherczeg/sljit/issues/89
     elif is_s390x; then
         "$CARGO" build --release --target=$TARGET
+    elif is_aarch64_musl; then
```

Comment:
would this never fall into the first condition?

Reply:
the targets are dependent on the architecture variable and you can see that aarch64 , aarch64-musl are distinct https://github.com/microsoft/ripgrep-prebuilt/blob/aa3a29dd462cd264dd4d5a302b8309660fc3df97/build/utils.sh#L22-L46

Reply:
You can check that the env is applied correctly at https://dev.azure.com/vscode/ripgrep-prebuilt/_build/results?buildId=95112&view=logs&j=29bcf809-c0df-5a9c-77e9-04c1800335da&t=9228d288-de8e-50d8-cf3b-6c357507fd2d